es/09_0_Introduciendo_Bitcoin_Scripts.md

```diff
@@ -0,0 +1,27 @@
+# Capítulo 9: Introducción a los scripts de Bitcoin
+
+Hasta la fecha, hemos estado interactuando con Bitcoin a un nivel de abstracción relativamente alto. El programa `bitcoin-cli` ofrece acceso a una variedad de comandos RPC que admiten la creación y el control de transacciones de Bitcoin sin procesar que incluyen fondos, datos, cronómetros y multisigs.
+
+Sin embargo, Bitcoin ofrece mucha más complejidad que eso. Incluye un lenguaje de programación simple que se puede utilizar para crear condiciones de canje aún más complejas. Si las multifirma y los bloqueos de tiempo proporcionaron las bases de los contratos inteligentes, entonces Bitcoin Script se cimienta en esa base. Es el siguiente paso para empoderar a Bitcoin.
+
+## Objetivos de este capítulo
+
+Después de trabajar en este capítulo, un desarrollador podrá:
+
+   * Diseñar un script de Bitcoin
+   * Aplicar un script de Bitcoin
+   
+Los objetivos de apoyo incluyen la capacidad de:
+
+   * Comprender el propósito de los scripts de Bitcoin
+   * Comprender el script P2PKH
+   * Comprender cómo funciona P2WPKH con secuencias de comandos
+   * Comprender las necesidades de las pruebas de scripts de Bitcoin
+   
+## Tabla de contenido
+
+* [Sección uno: Entendiendo la base de las transacciones](09_1_Entendiendo_la_base_de_las_Transacciones.md)
+* [Sección dos: Ejecución de un script de Bitcoin](09_2_Ejecutando_un_Script_Bitcoin.md)
```

Comment:
`[Sección dos: Ejecutando un Script Bitcoin](09_2_Ejecutando_un_Script_Bitcoin.md)`

- Further changes needed in #377.